runtime/experimental/helpers/export.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-cp /tmp/php-$(php -r 'echo phpversion();').zip /export/php-fpm-$(php -r 'echo phpversion();').zip
-cp /tmp/php-cli-$(php -r 'echo phpversion();').zip /export/php-default-$(php -r 'echo phpversion();').zip
```

Comment:
This file wasn't used anymore, removed it.

Reply:
Delete all the unused things! ;-)